### PR TITLE
MP-6835: added missing changes that prevented normal work of quick search widget

### PR DESCRIPTION
**Pull request by nPeretyatko**
Ticket: https://spryker.atlassian.net/browse/MP-6835
Changes: fixed work of product search widget in cart and shopping list pages.

### CODE REVIEW

File: src/Pyz/Yves/ProductSearchWidget/Theme/default/views/product-quick-add/product-quick-add.twig [hunk 1]
```diff
@@ -23,31 +23,4 @@
...
         }
     } only %}
-        {% block body %}
-            {% if data.title is not empty %}
-                <div class="col col--sm-12">
-                    <h2 class="{{ config.name }}__title title title--h5 spacing-bottom spacing-bottom--big {{ not data.verticalLayout ? config.name ~ '__title--row' }}">
-                        {{ data.title }}
-                    </h2>
-                </div>
-            {% endif %}
```
Comment (FRITS-Kh) [inline src/Pyz/Yves/ProductSearchWidget/Theme/default/views/product-quick-add/product-quick-add.twig:33]: What about the design changes, has it been approved by the PO?

**Review by FRITS-Kh — CHANGES_REQUESTED** [review]

File: src/Pyz/Yves/ProductSearchWidget/Theme/default/views/product-quick-add/product-quick-add.twig (hunk 1 above)
Comment (danilmoro3) [reply src/Pyz/Yves/ProductSearchWidget/Theme/default/views/product-quick-add/product-quick-add.twig:33]: This part was missed during integration from `b2b-demo-shop-internal`
https://github.com/spryker/b2b-demo-shop-internal/pull/1509/files#diff-eefa27467167226909d1dc6ae87b1bd6b836d09286bfc9075e98e8782e40dc3a

Comment (FRITS-Kh) [reply src/Pyz/Yves/ProductSearchWidget/Theme/default/views/product-quick-add/product-quick-add.twig:33]: > This part was missed during integration from `b2b-demo-shop-internal` https://github.com/spryker/b2b-demo-shop-internal/pull/1509/files#diff-eefa27467167226909d1dc6ae87b1bd6b836d09286bfc9075e98e8782e40dc3a

It doesn't mean that title and spaces should be changed during integration.

Comment (danilmoro3) [reply src/Pyz/Yves/ProductSearchWidget/Theme/default/views/product-quick-add/product-quick-add.twig:33]: All changes were done in the `product-quick-add-form.twig`, so this embed is redundant

**Review by FRITS-Kh — APPROVED** [review]

File: src/Pyz/Yves/ProductSearchWidget/Theme/default/views/product-quick-add/product-quick-add.twig (hunk 1 above)
Comment (FRITS-Kh) [reply src/Pyz/Yves/ProductSearchWidget/Theme/default/views/product-quick-add/product-quick-add.twig:33]: Didn't notice that the `product-quick-add-form` was extended at the project level.

**Pull request merged**